config/helm/aws-node-termination-handler/values.yaml

```diff
@@ -30,14 +30,13 @@ resources:
     memory: "64Mi"
     cpu: "50m"
   limits:
-    memory: "128Mi"
-    cpu: "100m"
```

Comment:
e.g. https://medium.com/omio-engineering/cpu-limits-and-aggressive-throttling-in-kubernetes-c5b20bd8a718

Reply:
This was specifically related to the Linux CFS scheduler bug which has been patched and back ported to 4.18.  aws-node-termination-handler doesn't require much in terms of cpu or memory anyways, so I doubt the bug would have impacted NTH. The limits could probably match the request tbh. If the CFS bug did or still can impact NTH, a pause wouldn't be that bad for NTH, since it's just responding to events and is required to act in seconds, not milliseconds.  The limits provide a nice safety mechanism in the case of a bug introduced in the codebase, so I don't think the cons of having it outweigh that benefit.